config/7.11.0/dashbuilder/rhpam-standalone-dashbuilder.yaml

```diff
@@ -47,6 +47,7 @@ dashbuilder:
               app: "[[.ApplicationName]]"
               application: "[[.ApplicationName]]"
               service: "[[.ApplicationName]]-[[.Dashbuilder.Name]]"
+              cluster: "jgrp.k8s.[[.ApplicationName]].[[.Dashbuilder.Name]]"
```

Comment:
please add unit tests that look for these labels... not just dashbuilder, but for all components. seems these labels are critical to operation so need coverage for future PRs.